scripts/test/lint.py

```diff
@@ -86,6 +86,20 @@ def run_cmd(cmd):
     return out
 
 
+class InvalidDescriptionLength(Lint):
+    max_length = 175
+    name = "the description length is invalid"
+    recommendation = "the description must be limited to 174 characters maximum."
+
+    def check(self, path):
+        dom = minidom.parse(str(path))
+        description = dom.getElementsByTagName("description")[0].firstChild.data
+        if not (len(description) <= self.max_length):
```

Comment:
change the condition so that you don't need the `not` making it easier to read:

```suggestion
        if len(description) > self.max_length:
```